### ISSUE TITLE
Bug: Dont work on Android 8

### ISSUE THREAD

**Issue by jordanks** [EddyVerbruggen/Calendar-PhoneGap-Plugin#424]
Hi, I think I have found a bug that happens only on Android 8 (testing on google pixel).
There is a problem with the plugin using the createEvent method:
1. It creates an event on the day but is tagged as "Birthday".
2. It is not possible to specify a time (or the time is ignored)
3. The event disappears when you open it.

I tried the same apk build on phones with Android 7 and 6 and the event is created as expected with the time correctly.

Is this a known issue?

**Comment by EddyVerbruggen** [comment]
Nope, I guess largely because the Pixel is the only phone running Android 8 atm. I have a shiny new Samsung S8 running 7.0 (not even 7.1) so can't help you (yet).

**Comment by alexrindone** [comment]
@jordanks and @EddyVerbruggen  this may not be an issue on Android 8. Are you using listCalendars() to get a list of calendars to set the calendar options with a calendar id? I ran into this issue and it was because events were trying to be created on contacts, and those events would default to birthdays. Use listCalendars(), get a calendar id, and then create the event. It'll work. There are about 3 or 4 other issues in here explaining this similar issue.

**Comment by terpro** [comment]
listCalendars -> check for isPrimary -> save to that calendar
https://github.com/EddyVerbruggen/Calendar-PhoneGap-Plugin/issues/363
